plugins/traction_innkeeper/traction_innkeeper/v1_0/innkeeper/tenant_manager.py

```diff
@@ -124,10 +127,19 @@ async def create_tenant(self, wallet_id: str, tenant_id: str = None):
         return tenant
 
     async def create_innkeeper(self):
-        config = self._config.innkeeper_wallet
+        config: InnkeeperWalletConfig = self._config.innkeeper_wallet
         tenant_id = config.tenant_id
         wallet_name = config.wallet_name
         wallet_key = config.wallet_key
+        # multi_ledger_manager = self._profile.inject(BaseMultipleLedgerManager)
+        self._profile.settings.set_value(
+            "tenant.endorser_config",
+            config.connect_to_endorser,
+        )
+        self._profile.settings.set_value(
+            "tenant.public_did_config",
+            config.create_public_did,
+        )
```

Comment:
Can these be specified per-tenant? If it is a setting at the InnKeeper wallet level, how do we specify for each tenant which one can have a public DID and/or become an issuer?

Reply:
Yeah the InnkeeperWallet naming here confused me as well at first :)